license.txt

```diff
@@ -0,0 +1,12 @@
+
+
+<!--
+Copyright (c) 2018 by freeCodeCamp (https://codepen.io/freeCodeCamp/pen/MJjpwO)
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
```

Comment:
- [ ] Add this page to your README file, as a link in the License section

Reply:
>  - [ ] Add this page to your README file, as a link in the License section
- [ ] Please 🙏try to always be as polite as possible. Remember we don't want the viewer to feel like we are ordering the 👍

Reply:
Great Job adding a License page! however

- [ ] I think it would be better if you add this page to your README file, as a link in the License section